int64 mod
float64 lin_x
float64 ang_z
float64 pos_x
float64 pos_y
float64 ang_theta
---
bool response
